source/module_lr/esolver_lrtd_lcao.cpp

```diff
@@ -326,13 +328,17 @@ LR::ESolver_LR<T, TR>::ESolver_LR(const Input_para& inp, UnitCell& ucell) : inpu
         psi_u,
         dpsi_u,
         d2psi_u,
-        GlobalV::NUM_STREAM);
-    psi_u.clear();
```

Comment:
why did you delete psi_u.clear() and the following lines?

Reply:
`psi_u` is created within this function. Since it is of vector type, it is automatically released when the function ends, so there is no need for manual memory management.